### ISSUE TITLE
Browsers cache lift.js even if source has changed.

### ISSUE THREAD

**Issue by joescii** [lift/framework#1755]
As pointed out in [this thread on the ML](https://groups.google.com/forum/#!topic/liftweb/VY10pnY0MCs), Lift 3's `lift.js` artifact will be cached by browsers.  I have verified that changes to `lift.js` do not in any way trigger a refresh of the asset in the browser.  This will cause browsers to use older version of `lift.js` as teams upgrade Lift versions.

**Comment by Shadowfiend** [comment]
We can, if we want to, make `lift.js` also alias as `lift-3.0.0.js`, so you can include the version in the `script` reference.

(The good news is, `lift.js` changes are (a) relatively rare and (b) usually internal.)

**Comment by joescii** [comment]
Yep, I certainly agree they're rare.  But rare things that need manual intervention are arguably more harmful than a frequent change requiring manual handling.... it's easy to forget!

**Comment by Shadowfiend** [comment]
For sure!
